spec/features/categories/user_edits_a_category_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+describe 'User edits a category' do
+  scenario 'user edits a category' do
```

Comment:
what if a user enters invalid data while editing? we should probably have a test for that!